recipes/nuraft/all/conanfile.py

```diff
@@ -18,27 +18,25 @@ class NuRaftConan(ConanFile):
     settings = "os", "compiler", "arch", "build_type"
 
     options = {
+        "asio": ["boost", "standalone"],
         "shared": ['True', 'False'],
         "fPIC": ['True', 'False'],
     }
     default_options = {
+        "asio": "boost",
         "shared": False,
         "fPIC": True,
     }
 
-    generators = "cmake", "cmake_find_package"
-
-    @property
-    def _source_subfolder(self):
-        return "source_subfolder"
-
     def export_sources(self):
-        self.copy("CMakeLists.txt")
         for p in self.conan_data.get("patches", {}).get(self.version, []):
             self.copy(p["patch_file"])
```

Comment:
Should be `copy(self, ...`

https://github.com/conan-io/conan-center-index/blob/624d6cfa702eecd4f0af9d32c1d33e4e409f7631/recipes/polylineencoder/all/conanfile.py#L29

Reply:
:+1: